server/webapp/WEB-INF/rails/webpack/single_page_apps/new_dashboard.js

```diff
@@ -224,6 +222,7 @@ $(() => {
     });
 
     renderView();
+    repeater().start();
```

Comment:
Seem to be creating a new repeater every time. Is that intentional?

Reply:
@ketan -- this is moved under `onPluginInfosResponse` which will be called only once after the dashboard page is loaded.